.github/workflows/actionlint.yml

```
@@ -1,18 +1,20 @@
 # GitHub Workflowsに関するプルリクエストに対し、reviewdogのlintを実行します。
 
-name: actionlint
+name: Actionlint
 
 on:
   pull_request:
     paths:
-      - '.github/workflows/**'
+      - '.github/workflows/**.yml'
+      - '.github/actions/**/**.yml'
```

globパターン的に書くならこうじゃないですか？
ついでにいうと、actionは`action.yml`で名前固定なのでワイルドカード指定は必要ないと思います。
```suggestion
      - '.github/workflows/*.yml'
      - '.github/actions/**/action.yml'
```